ext/adaptors/ranges-lib.tex

```diff
@@ -14,6 +14,7 @@
 
   // \ref{ranges.view_interface}:
   template <class D>
+    @\added{requires is_class_v<D>}@
```

Comment:
I don't see where the standard permits `is_class` to be instantiated with an incomplete type.

Reply:
https://timsong-cpp.github.io/cppwp/meta.type.synop#2